targets/TARGET_GigaDevice/TARGET_GD32F4XX/CMakeLists.txt

```diff
@@ -84,3 +84,8 @@ target_sources(mbed-core
 
         ${STARTUP_FILE}
 )
+
+mbed_set_linker_script(mbed-gd32-f450zi ${CMAKE_CURRENT_SOURCE_DIR}/${LINKER_FILE})
```

Comment:
```suggestion
mbed_set_linker_script(mbed-gd32f4xx ${CMAKE_CURRENT_SOURCE_DIR}/${LINKER_FILE})
```

Reply:
I belive this one is fine as it is. The final target is 450zi, the 4xx is just a parent.

Reply:
Ah wait, no it is other way around, also in the another target in this PR. I'll commit it now